waspc/data/Generator/templates/server/src/dbClient.js

```diff
@@ -0,0 +1,44 @@
+{{={= =}=}}
+import Prisma from '@prisma/client'
+
+import { hashPassword } from './core/auth.js'
+
+{=# userEntityUpper =}
+const PASSWORD_FIELD = 'password'
+
+{=/ userEntityUpper =}
+const createDbClient = () => {
+  const prismaClient = new Prisma.PrismaClient()
+
+  {=# userEntityUpper =}
+  prismaClient.$use(async (params, next) => {
```

Comment:
Wow all this is rather complicated hm! It was simpler in mongoose.

Reply:
yes, it isn't per-field but we rather have to capture and analyse every call.